### PR TITLE
adding health warning about insecure protocols for LOAD CSV

### PR DESCRIPTION
**Pull request by phil198**
adds a warning against allowing insecure protocols for `LOAD CSV`

this PR is the operations manual copy of the [cypher manual equivalent](https://github.com/neo4j/docs-cypher/pull/1006)

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1763.surge.sh

Updated pages:
- [Load privileges](https://neo4j-docs-operations-1763.surge.sh/operations-manual/5/authentication-authorization/load-privileges)

File: modules/ROOT/pages/authentication-authorization/load-privileges.adoc [hunk 1]
```diff
@@ -60,6 +60,14 @@ Unlike other privileges, the `LOAD` privilege is not granted, denied, or revoked
 Adding `ON ALL DATA` means a role has the privilege to load data from all sources.
 To only allow loading data from sources given by a specific CIDR range use `ON CIDR cidr`.
 
+[IMPORTANT]
+====
+It is strongly recommended to permit resource loading only over secure protocols such as HTTPS instead of insecure protocols like HTTP.
+This can be done by limiting the accessible ranges to only trusted sources that use secure protocols.
+If allowing an insecure protocol is absolutely unavoidable, Neo4j takes measures internally to enhance the security of these requests within their limitations.
+However, this means that insecure URLs on virtual hosts will not function unless you add the JVM argument `-Dsun.net.http.allowRestrictedHeaders=true` to the configuration setting link:{neo4j-docs-base-uri}/operations-manual/{page-version}/configuration/configuration-settings/#config_server.jvm.additional/[jvm.additional].
```
Comment (renetapopova) [inline modules/ROOT/pages/authentication-authorization/load-privileges.adoc:68]:
```suggestion
However, this means that insecure URLs on virtual hosts will not function unless you add the JVM argument `-Dsun.net.http.allowRestrictedHeaders=true` to the configuration setting xref:configuration/configuration-settings.adoc#config_server.jvm.additional[server.jvm.additional].
```

**Review by renetapopova — APPROVED** [review]
A minor suggestion.

File: modules/ROOT/pages/authentication-authorization/load-privileges.adoc (hunk 1 above)
Comment (phil198) [reply modules/ROOT/pages/authentication-authorization/load-privileges.adoc:68]: thank you

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1763.surge.sh

Updated pages:
- [Load privileges](https://neo4j-docs-operations-1763.surge.sh/operations-manual/5/authentication-authorization/load-privileges)

File: modules/ROOT/pages/authentication-authorization/load-privileges.adoc [hunk 2]
```diff
@@ -60,6 +60,14 @@ Unlike other privileges, the `LOAD` privilege is not granted, denied, or revoked
 Adding `ON ALL DATA` means a role has the privilege to load data from all sources.
 To only allow loading data from sources given by a specific CIDR range use `ON CIDR cidr`.
 
+[IMPORTANT]
+====
+It is strongly recommended to permit resource loading only over secure protocols such as HTTPS instead of insecure protocols like HTTP.
+This can be done by limiting the accessible ranges to only trusted sources that use secure protocols.
+If allowing an insecure protocol is absolutely unavoidable, Neo4j takes measures internally to enhance the security of these requests within their limitations.
+However, this means that insecure URLs on virtual hosts will not function unless you add the JVM argument `-Dsun.net.http.allowRestrictedHeaders=true` to the configuration setting xref:configuration/configuration-settings.adoc#config_server.jvm.additional[server.jvm.additional].
```
Comment (renetapopova) [inline modules/ROOT/pages/authentication-authorization/load-privileges.adoc:68]:
```suggestion
However, this means that insecure URLs on virtual hosts will not function unless you add the JVM argument `-Dsun.net.http.allowRestrictedHeaders=true` to the configuration setting xref:configuration/configuration-settings.adoc#config_server.jvm.additional[`server.jvm.additional`].
```

**Review by renetapopova — APPROVED** [review]
sorry, one more tiny thing

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1763.surge.sh

Updated pages:
- [Load privileges](https://neo4j-docs-operations-1763.surge.sh/operations-manual/5/authentication-authorization/load-privileges)

**Pull request merged**